### PR TITLE
Added Azure Blob Storage for 'Mods'

### PR DESCRIPTION
**Pull request by RedEchidnaUK**
# Proposed Changes

Currently 'Mods' have to be local, on a more traditional local server file share or an http web server. Unfortunately, none of those options worked for my client, as they are cloud first and didn't want to run a web server just to host mod files. So, I have added the ability to use Azure Blob Storage to host 'Mods'. The functionality has been added in a way to maintain backwards compatibility with the other 'Mods' options.

I have incremented the version to 1.16.5. Not sure if that is okay, but i needed to increase it to something for the ADMX files to reference when this feature was added.

### CODE REVIEW

**Comment by RedEchidnaUK** [comment]
@KnifMelti Happy to have helped out.
Despite how my body feels somedays, I'm not near 60 yet, but I am just about old enough to have come from a cmd scripting era.

File: Winget-AutoUpdate-Install.ps1 [hunk 1]
```diff
@@ -113,7 +117,7 @@ param(
 
 <# APP INFO #>
 
-$WAUVersion = "1.16.4"
+$WAUVersion = "1.16.5"
```
Comment (Romanitho) [inline Winget-AutoUpdate-Install.ps1:120]: As a new feature, we could even go for a 1.17.0 as pre-release ?

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1:120]: It's not my project anymore 😅 so anyone can do what he thinks good for all of us 😊

Comment (RedEchidnaUK) [reply Winget-AutoUpdate-Install.ps1:120]: @KnifMelti Do you have an opinion on this? I think 1.17.0 as a pre-release makes more sense as it is a new feature.

**Comment by KnifMelti** [comment]
Go for it!

_____

      KnifMelti
- Lädervirtuos -
No Average Craftsman

(N) +46.705.466304
(A) ***@***.*** www.knifmelti.se
(C) PGP Key - www.knifmelti.se/gaj.asc

Den mån 13 mars 2023 12:35Red Echidna UK ***@***.***> skrev:

File: Winget-AutoUpdate-Install.ps1 (hunk 1 above)
Comment (KnifMelti) [reply Winget-AutoUpdate-Install.ps1:120]: I beg to differ @Romanitho **Owner**!
It's very much **your** project and we're all very grateful that you took the time to initiate and code the very foundation and higher (in my view) functions/layouts we all (I specifically) depend upon ❤️
This came in a time when it was much needed and is now a part of many users **install-and-forget** applications that just works, both in corporate environments and in private.
So, don't you now go quitting on us!

But, compared to your new Daddy life it's a minor/smaller thing 😉

**Pull request merged**